### PR TITLE
Fixed #27158 -- Fixed positioning of admin's many-to-many widgets in rtl.css.

### PR DESCRIPTION
**Pull request by mehdipourfar**
Compare this two images to see what exactly this commit does:

http://uploads.im/0B6JX.jpg
http://uploads.im/J8AqP.jpg

### CODE REVIEW

**Comment by timgraham** [comment]
Could you please open a Trac ticket for this issue? Unless it's a regression from an earlier version of Django, it won't be backported to 1.10 (see our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions)).

File: django/contrib/admin/static/admin/css/widgets.css [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 .selector-chosen select {
     border-top: none;
+    height: 20.8em;
```
Comment (bmispelon) [inline django/contrib/admin/static/admin/css/widgets.css]: What's the significance of this value? It seems a bit arbitrary.

Comment (mehdipourfar) [inline django/contrib/admin/static/admin/css/widgets.css]: As you can see in snapshots above, it makes them vertically aligned.

Comment (timgraham) [inline django/contrib/admin/static/admin/css/widgets.css]: I think Baptiste's question was how did you calculate it. It looks like it's the sum of `.selector select { height: 17.2em; }` and some other height?

p.s. For clarity, I think this would be better as two separate commits.

Comment (bmispelon) [inline django/contrib/admin/static/admin/css/widgets.css]: The mismatched height issue is present in both LTR and RTL versions so we might want to treat it as a separate issue (or at least as two commits, as Tim said).

Having a comment explain how you came up with that oddly specific value would be helpful to future maintainers.

**Comment by bmispelon** [comment]
Maybe my lack of CSS knowledge is the problem, but I don't understand why `display: table` fixes the alignment issue here. This display style isn't used anywhere else in the admin (whether in rtl or ltr).

Also, it still introduces a slight vertical alignment issue:

(LTR version: the left side of the input is on the same vertical line as the left side of the filter)
add_participant_django_site_admin_-_2016-09-01_10 38 26

(RTL version: the right side of the input is not on the same vertical line as the right side of the filter)
_participant_ _ _ _-_2016-09-01_10 38 00

File: django/contrib/admin/static/admin/css/widgets.css (hunk 1 above)
Comment (mehdipourfar) [inline django/contrib/admin/static/admin/css/widgets.css]: To be honest, I've chosen 20.8em experimentally because with this height, they seem vertically aligned. There is no calculation behind it.

**Comment by mehdipourfar** [comment]
Yes. You are right. {display: table} represents some  issues here.
The better and cleaner solution is this.
```
.related-widget-wrapper {
    float: right;
}
form .aligned p.help {
    clean: right;
}
```
If you are all agree, i'll can create a new commit

**Comment by timgraham** [comment]
Looks okay at first glance.

**Comment by timgraham** [comment]
Hi, did you forget about this?

**Comment by mehdipourfar** [comment]
Hi, sorry, I didn't forget about it. But I'm a little bit confused about the way to do it.
Should I create a new branch from stable/1.10.x named 27158 and commit on that branch and send a pull request or can I use this branch for it? In latter case, should I remove my previous commits?

I haven't contributed to opensource projects before and it seems that you are really strict about the way of using git. That's great and I admire you.

If you try to help me, I'll promise to do more contributions after that:)

**Comment by timgraham** [comment]
The pull request should go to master. Since you can't change the target branch for a PR, I'll close this one and you can send a new one.

If it's not a regression from a previous release or a major bug in a new feature it won't be backported. See our [supported versions policy](https://docs.djangoproject.com/en/dev/internals/release-process/#supported-versions). (Even for items that are backported, we prefer the pull request to master and then the committer cherry-picks to the stable branch.)

Thanks!
